parser/src/python.rs

```diff
@@ -31184,13 +31185,15 @@ fn __action164<
             None => (vec![], vec![]),
         };
         let end_location = body.last().unwrap().end();
+        let type_params = Vec::new();
```

Comment:
Here we just use empty vectors, proper parsing and population of these fields will come in a follow-up.

Reply:
It seems improper to modify this without modifying the LALRPOP file, but I'm not sure it matters and it'd be nice to add parsing of each new syntax form in separate pull requests if feasible.

Reply:
Since this file will be overwritten on dev build step, this file is not expected to be manually edited.
You can find the same code in `python.lalrpop` file.

https://github.com/RustPython/Parser/blob/69d27d924c877b6f2fa5dc75c9589ab505d5b3f1/parser/src/python.lalrpop#L1133-L1135

Reply:
How to find the relevant line: I tend just to copy the code and search for it in the lalrpop file.